daemons/gptp/common/avbts_message.hpp

```diff
@@ -655,6 +688,72 @@ class FollowUpTLV {
 	int32_t getRateOffset() {
 		return cumulativeScaledRateOffset;
 	}
+
+    /**
+     * @brief  Updates the scaledLastGmFreqChanged private member
+     * @param  val Value to be set
+     * @return void
+     */
+    void setScaledLastGmFreqChange(int32_t val)
+    {
+        scaledLastGmFreqChange = PLAT_htonl(val);
+    }
+
+    /**
+     * @brief  Gets the current scaledLastGmFreqChanged value
+     * @return scaledLastGmFreqChange
+     */
+    int32_t getScaledLastGmFreqChange(void)
+    {
+        return scaledLastGmFreqChange;
+    }
+
+    /**
+     * @brief  Sets the gmTimeBaseIndicator private member
+     * @param  tbi Value to be set
+     * @return void
+     */
+    void setGMTimeBaseIndicator(uint16_t tbi)
+    {
+        gmTimeBaseIndicator = PLAT_htonl(tbi);
+    }
```

Comment:
This implementation conflicts with the inc method below. ie can't increment a multi-byte variable using ++ if it is stored in network order. I'm actually opposed to storing anything in network order, but that may be too large of a change at this point - and out of scope for this work.